ansible/cleanup.yml

```
@@ -0,0 +1,74 @@
+        name: nginx
+        state: restarted
+      ignore_errors: yes
+
+    - name: Clean up any Python cache
+      shell: |
+        find /root -name "__pycache__" -type d -exec rm -rf {} + 2>/dev/null || true
+        find /root -name "*.pyc" -delete 2>/dev/null || true
+
+    - name: Display cleanup completion
+      debug:
+        msg: "✅ VPS cleaned up and ready for fresh deployment!" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)